packages/ai/core/generate-text/index.ts

```diff
@@ -30,7 +30,6 @@ export type {
 export type { ToolCallRepairFunction } from './tool-call-repair';
 export type {
   CoreToolResult,
-  CoreToolResultUnion,
```

Comment:
why was this export removed? it's still useful: https://sdk.vercel.ai/docs/ai-sdk-core/tools-and-tool-calling#types

Reply:
oh nevermind it's core - but would be good to remove deprecated stuff in a separate pr

Reply:
good candidate for another pr:

![CleanShot 2025-04-11 at 15 50 47@2x](https://github.com/user-attachments/assets/d856c395-21f5-4b95-a984-a596e6a2c668)